Mathlib/MeasureTheory/Function/L1Space/Integrable.lean

```diff
@@ -1035,6 +1038,6 @@ lemma Integrable.snd {f : α → E × F} (hf : Integrable f μ) : Integrable (fu
 
 lemma integrable_prod {f : α → E × F} :
```

Comment:
Honest question: should this also be `prodMk`? (This can happen in a follow-up for sure.)

Reply:
I don't want to increase the number of renames in this PR. Also, `f` takes values in `E × F`, it isn't `fun x => (g x, h x)`.